app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+from typing import List
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills: List[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
```

Comment:
You're using `append` method to add a single value to a list (checklist point 1). It's not an error, but using `extend()` or `+=` with a list of one element would be more consistent with the rest of your code.